<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CategoriasDelete</class>
 <widget class="QDialog" name="CategoriasDelete">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="barraSeleccion">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layoutBotones">
    <item>
     <widget class="QPushButton" name="btnEliminarTodas">
      <property name="text">
       <string>Eliminar Todas</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnAceptar">
      <property name="text">
       <string>Aceptar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancelar">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEliminar">
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="seleccionaArchivo">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>90</y>
     <width>153</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Selecciona la categoria:</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layoutCabecera"/>
  </widget>
  <widget class="QLabel" name="msgImportante">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mensaje Importante</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
